deploy.sh

```diff
@@ -125,8 +125,13 @@ svn commit -m "Update to version $VERSION from GitHub" --no-auth-cache --non-int
 
 if $INPUT_GENERATE_ZIP; then
   echo "Generating zip file..."
-  cd "$SVN_DIR/trunk" || exit
-  zip -r "${GITHUB_WORKSPACE}/${SLUG}.zip" .
+
+  # use a symbolic link so the directory in the zip matches the slug
+  ln -s "${SVN_DIR}/trunk" "{$SVN_DIR}/${SLUG}"
+  mv "${SVN_DIR}/trunk" "${SVN_DIR}/${SLUG}"
```

Comment:
Thanks for the PR @JasonTheAdams.
@JasonTheAdams As we are already creating symlink on line 130, do we need to move files from `trunk` to the newly created folder? I think this will throw the error, as it's kind of move to the folder itself.

Reply:
@JasonTheAdams pinging you here again to see if you can respond to Dharmesh's question above?

Reply:
Double-check me, but I don't think we need to move it anymore since the symlink is setting the directory name for the resulting zip. Just download the zip and make sure the directory inside has the same name as the zip file. If that's working then this `mv` is not needed.